css/style.css

```diff
@@ -285,9 +285,12 @@ video {
 }
 
 .carrousel-container {
-  width: 100%;
-  margin: 0 auto;
+  margin-bottom: 20px;
+  overflow: hidden;
+  padding-top: 56.25%;
```

Comment:
I do understand the use of this, but please add a comment denoting what it does, since at a first glance it can seem it's just an arbitrary padding.

Reply:
@jusk02 I already edited the comment and added the link of the tutorial where it explains the solution I implemented but basically the key is to create an empty div and set this one with a padding top of X percentage to create responsive containers with a constant aspect ratio. Here a table with values:

aspect ratio  | padding-top value
--------------|----------------------
    1:1       |       100%
    16:9      |       56.25%
    4:3       |       75%
    3:2       |       66.66%
    8:5       |       62.5%

Also you need to set up the div with an overflow hidden and a position different from default, in this case relative so this way we asure the content will be positioned regarding to this one and won't be overflowed so the last thing we need to do is add a new div within set with position absolute to asure this won't be out of the flow of its father container and also set top, bottom, left and right to use all the available space of it.